e2e-tests/cypress/tests/integration/channels/collapsed_reply_threads/channel_notifications_spec.ts

```diff
@@ -273,6 +274,7 @@ describe('CRT Desktop notifications', () => {
             cy.postMessageAs({sender, message: 'a thread', channelId: dmChannel.id, rootId: ''}).as('rootPost');
 
             // # Get post id of message
+            // @ts-expect-error: reply should be of type PostMessageResp
```

Comment:
I think you should be able to specify the type of this by rewriting the next line to start with
```js
cy.get<PostMessageResp>('@rootPost').then(...);
```
I think that should let you specify the type of the object passed to the `.then` callback properly